pkg/ddevapp/settings_test.go

```diff
@@ -82,3 +93,174 @@ func TestWriteDrushConfig(t *testing.T) {
 	err = os.RemoveAll(dir)
 	assert.NoError(t, err)
 }
+
+// TestIncludeSettingsDdevInNewSettingsFile verifies that when no settings.php file exists,
+// a settings.php file is created that includes settings.ddev.php.
+func TestIncludeSettingsDdevInNewSettingsFile(t *testing.T) {
+	dir := testcommon.CreateTmpDir("")
+	err := os.MkdirAll(filepath.Join(dir, "sites/default"), 0777)
+	assert.NoError(t, err)
+
+	app, err := NewApp(dir, DefaultProviderName)
+	assert.NoError(t, err)
+
+	for appType, relativeSettingsLocations := range appTypeSettingsLocations {
+		app.Type = appType
+
+		relativeSettingsLocation := relativeSettingsLocations[0]
+		relativeSettingsDdevLocation := relativeSettingsLocations[1]
```

Comment:
This is the magic positional params I was talking about earlier, hard to read and understand. Again, no need to change, especially in this very simple situation. I just overall prefer approaches that give a named parameter (map, struct, whatever)